index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+<h1> I been feeling like a mothafuckin postman</h1>
+<script src="build/react.js"></script>
+    <script src="build/react-dom.js"></script>
```

Comment:
Where's the `build` directory coming from? Did you miss committing a package.json file?